gcp/appengine/frontend3/src/templates/list.html

```diff
@@ -2,51 +2,48 @@
 {% set active_section = 'vulnerabilities' %}
 
 {% macro table_header_cell(column_id, column_name, is_sortable, is_sorted, is_descending, hide_on_mobile) %}
-<span
-class="vuln-table-cell mdc-data-table__header-cell vuln-table-header
+<span class="vuln-table-cell mdc-data-table__header-cell vuln-table-header
```

Comment:
Are all the changes in this file formatting changes? It's a little hard to tell what changed.

Reply:
I dropped a comment on the line of interest.